docs/design/ovs-pipeline.md

```diff
@@ -183,1015 +269,1363 @@ spec:
   appliedTo:
     - podSelector:
         matchLabels:
-          app: server
+          app: nginx
   ingress:
-    - action: Drop
+    - action: Allow
       from:
         - podSelector:
             matchLabels:
-              app: notClient
+              app: nginx
       ports:
         - protocol: TCP
           port: 80
+    - action: Drop
   egress:
     - action: Allow
       to:
         - podSelector:
             matchLabels:
-              app: dns
+              app: nginx
       ports:
-        - protocol: UDP
-          port: 53
+        - protocol: TCP
+          port: 80
+    - action: Reject
+```
+
+This ACNP is applied to all Pods with the `app: nginx` label in all Namespaces. For these Pods, it only allows TCP traffic
+on port 80 from all Pods with the label `app: nginx` and drop others. In addition to the ingress rules, this policy only
+allows egress TCP traffic on port 80 to all Pods with the label `app: nginx` and reject others. Similar to Kubernetes
+NetworkPolicy, Antrea will only install OVS flows for this policy on Nodes for which some of the Pods are the target of
+the policy. We still use `app:nginx` Pods mentioned above as the target of this policy.
+
+## Antrea L7 NetworkPolicy Implementation
+
+In addition to layer 3 and layer 4 policies mentioned above, [Antrea L7 NetworkPolicy](../antrea-l7-network-policy.md)
+is also supported in Antrea. The only difference is that Antrea L7 NetworkPolicy uses L7 protocol to match traffic, not
```

Comment:
I would say the "main difference", not the "only difference"

In particular (and you don't need to mention it here), the action of the L7 rule can only be Allow, and if the traffic is sent to the L7 engine for evaluation and the rule doesn't match, it will be dropped (subsequent policy rules will not be evaluated).